### ISSUE TITLE
Java SIGBUS fatal crash on `boot project --develop`

### ISSUE THREAD

**Issue by theronic** [degree9/meta#21]
Probably ran out of memory, but got this on first run:

```
atform.dylib+0x5fcc]  _platform_memmove$VARIANT$Haswell+0x12c

pm WARN boot-npm@0.1.0 No description
npm WARN boot-npm@0.1.0 No repository field.
npm WARN boot-npm@0.1.0 No license field.

Process completed successfully...

Starting file watcher (CTRL-C to quit)...

Loading project files...
• app/client.mustache...
• app/index.mustache...
• app/server.mustache...
• app/services.mustache...
• app/routing.mustache...
Loading project templates...
• app/client.cljs...
• app/index.cljs...
• app/server.cljs...
• app/services.cljs...
• app/routing.cljs...
Writing HTML files...
• index.html
Generating EDN file...
• nodejs.cljs.edn
Compiling ClojureScript...
• index.html.js
• nodejs.js
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGBUS (0xa) at pc=0x00007fffd4c09fcc, pid=87107, tid=0x0000000000009817
#
# JRE version: Java(TM) SE Runtime Environment (8.0_141-b15) (build 1.8.0_141-b15)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.141-b15 mixed mode bsd-amd64 compressed oops)
# Problematic frame:
# C  [libsystem_platform.dylib+0x5fcc]  _platform_memmove$VARIANT$Haswell+0x12c
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /Users/petrus/Projects/app/hs_err_pid87107.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
[1]    87107 abort      boot project --develop
 ~/Projects/app  boot project --develop
```

**Comment by flyboarder** [comment]
Yeah this is a memory issue, will be sorted in next version.

**Comment by theronic** [comment]
Any suggested workarounds?

**Comment by flyboarder** [comment]
not really, how much memory is in your dev rig? I suggest 16GB+

**Comment by flyboarder** [comment]
@theronic this should be solved in the latest snapshot `0.3.0-SNAPSHOT`

**Comment by DVious** [comment]
I tried to install meta, again, but in High Sierra - this problem no longer occurred.

No - now I have a new problem!

**Comment by flyboarder** [comment]
@DVious what is the new issue? Please open a new issue.

**Comment by DVious** [comment]
I'm still investigating whether to raise it as an issue but, I'll raise it, just now, and welcome views on it.

**Comment by theronic** [comment]
To my knowledge I was on High Sierra at the time I opened this, but I'm not 100% sure.
